app/auth/login/gallery/features/form/components/FormInput.tsx

```diff
@@ -10,14 +10,15 @@ import FormActions from "./FormActions";
 type Form = {
   email: string;
   password: string;
+  ip: string;
 };
 
-export default function FormInput() {
+export default function FormInput({ ip }: { ip: string }) {
   const router = useRouter();
 
   const [setIsLoading] = galleryLoginStore((state) => [state.setIsloading]);
 
-  const [form, setForm] = useState<Form>({ email: "", password: "" });
+  const [form, setForm] = useState<Form>({ email: "", password: "", ip });
```

Comment:
pass the ip directly into the signIn second arg

Copying props to state is an antipattern in react